src/AltinnCore/Runtime/Startup.cs

```diff
@@ -47,15 +47,27 @@ public Startup(IConfiguration configuration)
     /// <param name="services">The services available for asp.net Core</param>
     public void ConfigureServices(IServiceCollection services)
     {
+
+      string runtimeMode = string.Empty;
+      if (Environment.GetEnvironmentVariable("GeneralSettings__RuntimeMode") != null)
+      {
+          runtimeMode = Environment.GetEnvironmentVariable("GeneralSettings__RuntimeMode");
+      }
+      else
+      {
+          runtimeMode = Configuration["GeneralSettings:RuntimeMode"];
+      }
+
+
       // Adding services to Dependency Injection TODO: Make this environment specific
       bool loadFromPackage = false;
```

Comment:
No longer used? Can it be deleted?

Reply:
Se that its used on line 149, should it be changed on line 149 to use configuration value instead?